src/toolbar.cpp

```diff
@@ -154,6 +154,9 @@ bool GraphicsWindow::ToolbarDrawOrHitTest(int mx, int my,
 
     bool withinToolbar =
         (mx >= aleft && mx <= aright && my <= atop && my >= abot);
+    
+    // Initialize/clear menuHit.
+    if(menuHit) *menuHit = Command::NONE;
```

Comment:
You've moved this above the early return. Are you sure this is sound?

Reply:
Nevermind, I misunderstood.

Reply:
Yes, it is the early return that triggers the uninitialized memory error.